{"name": "test1_tap_search", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\nNoSuchElementError: An element could not be located on the page using the given search parameters.\n    at XCUITestDriver.doNativeFind (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:143:13)\n    at processTicksAndRejections (node:internal/process/task_queues:95:5)\n    at XCUITestDriver.findNativeElementOrElements (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:97:12)\n    at XCUITestDriver.findElOrEls (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:24:14)\n    at XCUITestDriver.findElOrElsWithProcessing (/Users/57block/.appium/node_modules/@appium/base-driver/lib/basedriver/commands/find.ts:60:12)\n    at XCUITestDriver.findElement (/Users/57block/.appium/node_modules/@appium/base-driver/lib/basedriver/commands/find.ts:75:12)", "trace": "self = <test_search.TestHomeSearch testMethod=test1_tap_search>\n\n    def setUp(self) -> None:\n        self.driver = get_driver()\n        global do\n        do = Actions(self.driver)\n>       do.login_flow(test_account)\n\ntestcase/home_page/test_search.py:17: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nutils/mobile_user_actions.py:884: in login_flow\n    self.input_password(account.password)\nutils/mobile_user_actions.py:52: in input_password\n    get_element(self.driver, password).clear().send_keys(password_type)\nutils/find_element.py:12: in get_element\n    raise e\nutils/find_element.py:6: in get_element\n    element = WebDriverWait(driver, wait_time).until(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"52345af2-e4a4-445f-a152-c8354fc3c75c\")>\nmethod = <function get_element.<locals>.<lambda> at 0x10abc4670>, message = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       NoSuchElementError: An element could not be located on the page using the given search parameters.\nE           at XCUITestDriver.doNativeFind (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:143:13)\nE           at processTicksAndRejections (node:internal/process/task_queues:95:5)\nE           at XCUITestDriver.findNativeElementOrElements (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:97:12)\nE           at XCUITestDriver.findElOrEls (/Users/57block/.appium/node_modules/appium-xcuitest-driver/lib/commands/find.js:24:14)\nE           at XCUITestDriver.findElOrElsWithProcessing (/Users/57block/.appium/node_modules/@appium/base-driver/lib/basedriver/commands/find.ts:60:12)\nE           at XCUITestDriver.findElement (/Users/57block/.appium/node_modules/@appium/base-driver/lib/basedriver/commands/find.ts:75:12)\n\nvenv/lib/python3.9/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException"}, "start": 1691400157665, "stop": 1691400178389, "uuid": "fecc3f6f-b7a0-42ef-a350-a11798b8afac", "historyId": "23757698f46a77308eb677aa7a06d87a", "testCaseId": "23757698f46a77308eb677aa7a06d87a", "fullName": "testcase.home_page.test_search.TestHomeSearch#test1_tap_search", "labels": [{"name": "story", "value": "get to search page"}, {"name": "feature", "value": "Home search"}, {"name": "severity", "value": "critical"}, {"name": "parentSuite", "value": "testcase.home_page"}, {"name": "suite", "value": "test_search"}, {"name": "subSuite", "value": "TestHomeSearch"}, {"name": "host", "value": "Joys-MacBook-Pro.local"}, {"name": "thread", "value": "6750-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.home_page.test_search"}]}